__tests__/test-utils.js

```diff
@@ -228,8 +228,23 @@ class TestUtils {
     });
   }
 
+  static startServer() {
+    return new Promise((resolve, reject) => {
+      const app = createApp();
+      server = app.listen(8088, err => {
+        if (err) {
+          reject(err);
+        }
+
+        request = supertest.agent(server);
```

Comment:
```suggestion
        // Ensures the supertest agent will be bound to the correct express port
        request = supertest.agent(server);
```
To understand why we're using this way of implementing the supertest would require a git blame or a search on the supertest's underlying agent API. I think a simple comment, as suggested above, would make it easier for a reader to understand why we chose this implementation.

Reply:
Done https://github.com/HathorNetwork/hathor-wallet-headless/pull/306/commits/652c02ab2bc917f8b2b6550bcdbbef883f1e60aa